<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuildingDialog</class>
 <widget class="QDialog" name="BuildingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="selectTypegroupBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>261</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="hqRadioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>HeadQuarters</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="outpostRadioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Outpost</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="farmRadioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Farm</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="mineRadioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Mine</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="tuniTowerRadioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>TuniTower</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="supplyChainRadioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>SupplyChain</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="infoLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose building type:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BuildingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BuildingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
